### ISSUE TITLE
(meta-)bicategories

### ISSUE THREAD

**Issue by jonsterling** [jonsterling/math#25]
I have been thinking a bit about how to deal with strictness and weakness in these notes. I want to avoid getting too arcane and religious about things, but at the same time I need just enough to set up the theory properly.

Consequently I think that it makes sense to extend and adjust the `Foundational Assumptions` section to define the concept of a "(meta-)bicategory", and then say that `CAT` is not the metacategory of all categories but rather the meta-bicategory of all categories. This is honestly needed already in order to provide intuition for what a cartesian fibration is.

Then later on under internal categories, we can define for any category `C` the *category* `Cat(C)` of categories internal to `C`. In particular when `C = Set`, this gives us an appropriate basis for the theory of strict categories, split fibrations, strictly discrete fibrations, etc.

We can get get a good rosetta stone now:

1. Cartesian fibration = pseudofunctor `C^op -> CAT`
2. Split cartesian fibration = functor `C^op -> Cat(SET)`
3. Right fibration = pseudofunctor `C^op -> GPD`
4. Discrete right fibration = pseudofunctor `C^op -> 0-Trunc(GPD)`
5. Strictly discrete right fibration = functor `C^op -> SET`.

@cangiuli What do you think about this?

**Comment by jonsterling** [comment]
For actually defining the strict discrete right fibrations, I would like to treat it as a structure on a discrete right fibration, much like a split cleavage is a structure on a fibration.

**Comment by cangiuli** [comment]
Yes, this is a great idea! Like the current treatment of meta-categories, I think it's worth not getting bogged down in the details, but this additional perspective is essential.

**Comment by cangiuli** [comment]
omg, "once we fix #25" does not mean the same thing as "fix #25" :facepalm:
